substrate/client/network/src/litep2p/mod.rs

```diff
@@ -166,8 +166,15 @@ pub struct Litep2pNetworkBackend {
 	/// Discovery.
 	discovery: Discovery,
 
-	/// Number of connected peers.
-	num_connected: Arc<AtomicUsize>,
+	/// Number of connected peers over the block announce protocol.
+	///
+	/// This is used to update metrics and network status.
+	num_sync_connected: Arc<AtomicUsize>,
```

Comment:
Looks like a metric that should be moved to the sync crate.

Reply:
Yep that sounds good, have created an issue for this:
- https://github.com/paritytech/polkadot-sdk/issues/5024

Since this was a bit involved for the peerstore metrics :D